File: Daily Challanges/2026-03-March/2026-03-26.py
def get_movie_night_cost(day, showtime, number_of_tickets):
    total_cost = 0
    ticket_cost = 12.00 if day in {"Friday", "Saturday", "Sunday"} else 10.00
    hour, _ = map(int, showtime[:-2].split(':'))
    meridiem = showtime[-2:]
    if not (hour >= 5 and meridiem == "pm") and day != "Tuesday":
        ticket_cost -= 2.00
    if day == "Tuesday":
        total_cost = number_of_tickets * 5.00
    else:
        total_cost = ticket_cost * number_of_tickets
    return f"${total_cost:.2f}"

print(get_movie_night_cost("Saturday", "10:00pm", 1))
print(get_movie_night_cost("Tuesday", "7:20pm", 2))
print(get_movie_night_cost("Wednesday", "5:40pm", 3))

File: Daily Challanges/2026-03-March/2026-03-26.js
function getMovieNightCost(day, showtime, numberOfTickets) {
  let ticketCost = ["Friday", "Saturday", "Sunday"].includes(day) ? 12.0 : 10.0;
  let hour = Number(showtime.split(":")[0]);
  const meridiem = showtime.slice(-2);
  if (meridiem === "pm") {
    hour += 12;
  }
  if (hour < 17 && day !== "Tuesday") {
    ticketCost -= 2.0;
  }

  if (day === "Tuesday") {
    ticketCost = 5.0;
  }
  const totalCost = ticketCost * numberOfTickets;
  return `$${totalCost.toFixed(2)}`;
}

console.log(getMovieNightCost("Saturday", "10:00pm", 1));
